config/admissions.uiowa.edu/config_split.config_split.admissions.yml

```diff
@@ -20,13 +20,45 @@ graylist:
   - core.entity_form_display.node.transfer_tips.default
   - core.entity_form_display.paragraph.admissions_card.default
   - core.entity_form_display.paragraph.admissions_requirement.default
+  - core.entity_view_display.node.area_of_study.default
```

Comment:
This seems to undo the clean up that @pyrello did in https://github.com/uiowa/uiowa/commit/639c8f21e1c64399c494058930d7ce8eeb506abc#diff-0d6789e2b54025afe51969cefc90c6014041f181116fe88518838b424bc3219d using wildcards to cover these.

Reply:
@joewhitsitt you know it's a problem when I can't come up with an explanation for this... 😬